anatomy/tracks/config-json.md

```diff
@@ -1 +0,0 @@
-This file has moved [here](https://github.com/exercism/docs/blob/main/building/tracks/config-json.md)
```

Comment:
Please don't remove this file, as it is there to gracefully handle old links.

Reply:
Removal was reverted